qemu/tests/cfg/blockdev_mirror_to_rbd.cfg

```diff
@@ -0,0 +1,29 @@
+- blockdev_mirror_to_rbd:
+    only Linux
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_to_rbd
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    target_images = mirror1
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    backup_options_data1 = sync
+    sync = full
+    storage_pools = default
+    storage_pool = default
```

Comment:
storage_pools = local remote
storage_type_remote = ceph
storage_type_local = directory
storage_pool_data1 = local 
storage_pool_mirror1 = remote

Here we add two storage pools: local and remote, which are mapping to the directory and rbd pools

Reply:
Have updated, thanks zhenchao.